src/inverseRobot.js

```diff
@@ -8,6 +8,18 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
```

Comment:
Make sense to create this variable after first return as in case of return you will not need it